pkgs/development/python-modules/glfw/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitHub, glfw3 }:
+
+buildPythonPackage rec {
+  pname = "glfw";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "FlorianRhiem";
+    repo = "pyGLFW";
+    rev = "v${version}";
+    sha256 = "1ygcwnh0x07yi87wkxykw566g74vfi8n0w2rzypidhdss14x3pvf";
+  };
+
+  patches = [ ./search-path.patch ];
+  postPatch = ''
+    substituteInPlace glfw/library.py --replace "@GLFW@" '${glfw3}/lib'
+  '';
+  propagatedBuildInputs = [ glfw3 ];
+
+  doCheck = false;
```

Helps with future maintenance aka the next person could simply check if there are now test available :wink: 

```suggestion
  # Project has no tests
  doCheck = false;
```

---

Thanks, I applied it.